skimage/exposure/tests/test_exposure.py

```diff
@@ -324,7 +324,7 @@ def test_adapthist_grayscale():
     adapted = exposure.equalize_adapthist(img, kernel_size=(57, 51),
                                           clip_limit=0.01, nbins=128)
     assert img.shape == adapted.shape
-    assert_almost_equal(peak_snr(img, adapted), 102.078, 3)
+    assert_almost_equal(peak_snr(img, adapted), 100.158, 3)
```

Comment:
why would the value change here?

Reply:
As mentioned in the description, the PR fixes border artefacts which are gone here but present in master. This would be reflected in this value no?

Reply:
By contrast, in the test `test_adapthist_alpha`, where the image size is a multiple of the kernel size and both the nd branch and master don't exhibit artefacts, values stay the same.